usecases/auth/authorization/conv/casbin_types.go

```diff
@@ -47,12 +47,13 @@ var (
 		authorization.Root:   CRUD,
 	}
 	actions = map[string]string{
```

Comment:
this is just used for printing, right? If yes, could we rename it? :) Was a bit confused at first what this is doing?

Reply:
yeah was thinking the same, but didn't wanna make it in the same PR, let me do it then